Start of documentation and example scripts (#1)
arfc/msr-spatial-dep: README.md

# msr-spatial-dep

> A collection of scripts associated with investigating spatial effects on molten salt reactor depletion.

Reviewer: ```suggestion
A collection of scripts for investigating the spatial effects on molten salt reactor depletion.
```

## Theory

> A brief discussion on the theory associated with this work is given in the `docs` directory.

Reviewer: ```suggestion
A brief discussion on the theory associated with this work is given in `docs/theory.md`.
```

## Preliminary Work

> Some preliminary work displaying a simplified version is given in the `scripts` directory under the `morty` sub-directory.
> This work analyzes single nuclides and five nuclides in the 135 isobar to see the effects of spatial tracking on xenon-135.

Reviewer: ```suggestion
Some preliminary work analyzes spatial tracking for xenon-135 in the 135 isobar for both a single nuclide and five nuclides. This work may be found in `scripts/morty/`.
```
